src/UbbIssBundle/Resources/views/Student/showGrades.html.twig

```diff
@@ -1,17 +1,48 @@
-{% extends "UbbIssBundle:layout.html.twig" %}
+<!DOCTYPE html>
+<html>
```

Comment:
Nu avem nevoie de <!DOCTYPE html> .... este in layout. si nici de <head>...
Incerca sa adaugi numa in blocku de stylesheet un css nou. Ca daca vezi ti-a disparut si header-u care trebuie sa fie pe toate paginile :)